pages/memorials.vue

```diff
@@ -0,0 +1,422 @@
+<template>
```

Comment:
This page seems to have been added in accidentally. I don't remember it being there before. Are you able to remove this as I assume it will be added in a different PR. Thanks Ruby

Reply:
oops, done!

Reply:
Looks good now. Thanks Ruby